src/main/java/frc/robot/Constants.java

```diff
@@ -47,6 +47,48 @@ public class Field {
         public static final FieldArea2d FRIENDLY_ALLIANCE_AREA = new FieldArea2d(0, 0, 5.85, 8.21);
     }
 
+    public class Shooter {
+        public static final double OUTTAKE_SPEED = .35;
+        public static final double INTAKE_SPEED = Undertaker.INTAKE_SPEED * .10;
+
+        public static final int INDEXER_MOTOR_ONE = 18;
+        public static final int INDEXER_MOTOR_TWO = 19;
```

Comment:
Change these to TOP and BOTTOM with respect to the robot in normal speaker shooting position